libraries/lib-xml/XMLWriter.cpp

```diff
@@ -459,11 +459,17 @@ void XMLUtf8BufferWriter::EndTag(const std::string_view& name)
    }
 }
 
+void XMLUtf8BufferWriter::WriteAttr(const std::string_view& name, const wxString& value)
+{
+   const auto utf8 = value.ToUTF8();
```

Comment:
ToUTF8 used here, utf8_str used instead in the following function -- made me check in the wxWidgets headers to be sure they were not really different.

Why not simply use utf8_str as in the existing function making the differences between the two functions less?